.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
> Actually, I'm talking about using the UMD build in a browser

You make me realize that `build:babel` is runned without `NODE_ENV=production` and that `build:min` isn't shipped into released. There is quite some stuff to change. 

> an AMD environment (RequireJS).

I'm not sure we can do much about that target.